libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKMapper.kt

```diff
@@ -452,10 +459,65 @@ fun readableMapOf(lspInformation: LspInformation): ReadableMap {
             "feeRate" to lspInformation.feeRate,
             "timeLockDelta" to lspInformation.timeLockDelta,
             "minHtlcMsat" to lspInformation.minHtlcMsat,
-            "channelFeePermyriad" to lspInformation.channelFeePermyriad,
             "lspPubkey" to lspInformation.lspPubkey,
-            "maxInactiveDuration" to lspInformation.maxInactiveDuration,
-            "channelMinimumFeeMsat" to lspInformation.channelMinimumFeeMsat
+            "openingFeeParamsList" to readableMapOf(lspInformation.openingFeeParamsList)
+    )
+}
+
+fun readableMapOf(openingFeeParams: OpeningFeeParams?): ReadableMap? {
+    if (openingFeeParams != null) {
+        return readableMapOf(
+                "minMsat" to openingFeeParams.minMsat,
+                "proportional" to openingFeeParams.proportional,
+                "validUntil" to openingFeeParams.validUntil,
+                "maxIdleTime" to openingFeeParams.maxIdleTime,
+                "maxClientToSelfDelay" to openingFeeParams.maxClientToSelfDelay,
+                "promise" to openingFeeParams.promise
+        )
+    }
+    return null
+}
+
+fun readableMapOf(openingFeeParamsMenu: OpeningFeeParamsMenu): ReadableMap {
+    return readableMapOf(
+            "values" to readableArrayOf(openingFeeParamsMenu.values)
+    )
+}
+
+fun readableMapOf(receiveOnchainRequest: ReceiveOnchainRequest): ReadableMap {
+    return readableMapOf(
+            "openingFeeParams" to readableMapOf(receivePaymentRequest.openingFeeParams)
+    )
+}
+
+fun readableMapOf(receivePaymentRequest: ReceivePaymentRequest): ReadableMap {
+    return readableMapOf(
+            "amountSats" to receivePaymentRequest.amountSats,
+            "description" to receivePaymentRequest.description,
+            "preimage" to receivePaymentRequest.preimage,
+            "openingFeeParams" to readableMapOf(receivePaymentRequest.openingFeeParams)
+    )
+}
+
+fun readableMapOf(receivePaymentResponse: ReceivePaymentResponse): ReadableMap {
+    return readableMapOf(
+            "lnInvoice" to readableMapOf(receivePaymentResponse.lnInvoice),
+            "openingFeeParams" to readableMapOf(receivePaymentRequest.openingFeeParams),
+            "openingFeeMsat" to receivePaymentRequest.openingFeeMsat
+    )
+}
+
+fun readableMapOf(buyBitcoinRequest: BuyBitcoinRequest): ReadableMap {
+    return readableMapOf(
+            "provider" to asBuyBitcoinProvider(buyBitcoinRequest.provider),
+            "openingFeeParams" to readableMapOf(buyBitcoinRequest.openingFeeParams)
+    )
+}
```

Comment:
This is not needed, but an asBuyBitcoinRequest function should be added to convert a ReadableMap to BuyBitcoinRequest.

Reply:
Done in 3b9ef403befd5079c2e761d6dacd0f0beda0e125